thinc/layers/tensorflowwrapper.py

```diff
@@ -32,18 +86,40 @@ def TensorFlowWrapper(
     if not isinstance(tensorflow_model, tf.keras.models.Model):
         err = f"Expected tf.keras.models.Model, got: {type(tensorflow_model)}"
         raise ValueError(err)
+
+    # Determine if the model is Sequential/Functional
+    is_subclass = False
+    try:
+        tensorflow_model.to_json()
+    except NotImplementedError:
+        is_subclass = True
+
+    if is_subclass:
+        for prop_name in ["catalogue_name", "eg_x", "eg_y", "eg_shape"]:
+            if not hasattr(tensorflow_model, prop_name):
+                raise ValueError(
+                    "Keras subclassed models are not whole-model serializable by "
+                    "Tensorflow. To work around this, you must decorate your keras "
```

Comment:
Ok, I take that "worst nitpick" comment back, this one is it 😂

```suggestion
                    "TensorFlow. To work around this, you must decorate your keras "
```

Reply:
Ahh right! I think I was got used to capitalizing it for the wrapper class names. Should I rename those?

Reply:
I think the class/function names are all fine? We try to use spelling consistent with the "official" spelling, so `TensorFlow` is correct.

Reply:
Oh, I had it wrong! 😅

Reply:
I can't accept your suggestion because I don't have write privs to thinc, so I applied your suggestion manually